libsql-ffi/build.rs

```diff
@@ -38,7 +38,14 @@ fn main() {
     };
 
     let dir = env!("CARGO_MANIFEST_DIR");
-    std::fs::copy(format!("{dir}/{bindgen_rs_path}"), &out_path).unwrap();
+
+    Command::new("cp")
+        .arg("--no-preserve=mode,ownership")
+        .arg("-R")
+        .arg(format!("{dir}/{bindgen_rs_path}"))
+        .arg(&out_path)
+        .output()
+        .unwrap();
```

Comment:
I am fairly sure this won't work for Windows builds. @LucioFranco I see we have some Windows checks as part of the CI so I wonder why they didn't catch this?

Reply:
windows just checks a very small feature subset, as we don't really support windows for the full libsql c code iirc.